SQL.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 const inspect = Symbol.for('nodejs.util.inspect.custom')
 const wrapped = Symbol('wrapped')
+const SQLSymbol = Symbol('SQL')
```

Comment:
I guess this should be named SqlStatementSymbol if we're going with this approach